### PR TITLE
Add quest to determine the availability of a toilet

### PR DESCRIPTION
**Pull request by ENT8R**
This PR adds a new quest to determine the availability of toilets in `mall` and `department_store`. It looks like this:

This time, the quest does even have an icon 😉 
I hope you like it 😄

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -385,4 +385,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_parkingType_surface">Normal</string>
     <string name="quest_parkingType_underground">Underground garage</string>
     <string name="quest_parkingType_multiStorage">Car park</string>
+    <string name="quest_toiletAvailability_title">Does this place have a toilet?</string>
```
Comment (westnordost) [inline app/src/main/res/values/strings.xml]: No, please require a name. How would the user know what "this place" is? Your screenshot is a good example of this, as the mall is not even a building outline.

**Comment by westnordost** [comment]
Also, generally, to be honest, to create quests, very little knowledge of Android or Java is necessary. So, many others can contribute these quests. If you are versed with Android, there are tons of more challenging things to do.

**Comment by ENT8R** [comment]
> Also, generally, to be honest, to create quests, very little knowledge of Android or Java is necessary. So, many others can contribute these quests. If you are versed with Android, there are tons of more challenging things to do.

Yes, I know. This should be my last PR with a new quest...

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (ENT8R) [inline app/src/main/res/values/strings.xml]: This is now implemented!

File: app/src/main/java/de/westnordost/streetcomplete/quests/toilet_availability/AddToiletAvailability.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddToiletAvailability extends SimpleOverpassQuestType
+{
+	@Inject public AddToiletAvailability(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		return "nodes, ways with (shop ~ mall|department_store and name) and !toilet";
```
Comment (westnordost) [inline app/src/main/java/de/westnordost/streetcomplete/quests/toilet_availability/AddToiletAvailability.java]: superfluous `(  )`

Comment (ENT8R) [inline app/src/main/java/de/westnordost/streetcomplete/quests/toilet_availability/AddToiletAvailability.java]: This is fixed now! Thank you for the hint!

Comment (westnordost) [inline app/src/main/java/de/westnordost/streetcomplete/quests/toilet_availability/AddToiletAvailability.java]: (It wasn't a problem, that is why I approved the PR already. Superfluous parenthesis are automatically removed)

**Pull request merged**